src/components/m-header-mobile/index.scss

```diff
@@ -87,24 +87,32 @@
   // https://github.com/axa-ch/patterns-library/issues/263
   background: $color-prim-white;
 
-  > .m-button {
-    display: block;
-    margin: $header-mobile__item-margin-top $header-mobile__item-padding-right 0 $header-mobile__item-padding-left;
-
-    text-align: center;
+  .m-header-top-content-bar__box {
+    padding-left: $header-mobile__item-padding-left;
+    padding-right: $header-mobile__item-padding-right;
+  }
 
+  axa-button {
     &:first-child {
-      margin-top: 0;
+      .m-button {
+        margin-top: 0;
+      }
     }
+  }
+}
 
-    + * {
-      margin-top: $header-mobile__item-margin-top;
-    }
+.m-button.m-header-mobile__box--button {
+  display: block;
+  margin: $header-mobile__item-margin-top $header-mobile__item-padding-right $header-mobile__item-margin-top $header-mobile__item-padding-left;
+
+  text-align: center;
+
+  + * {
+    margin-top: $header-mobile__item-margin-top;
   }
 
-  .m-header-top-content-bar__box {
-    padding-left: $header-mobile__item-padding-left;
-    padding-right: $header-mobile__item-padding-right;
+  @include respond-up(sm) {
+    width: $header-mobile__canvas-width-sm - $header-mobile__item-padding-right - $header-mobile__item-padding-left;
```

Comment:
hmh is the width changing here?
I'm asking because this was inspected very carefully by Anja and approved.

Reply:
it should not look different than before, but as now there is a DOM element in between (`axa-button`) it have to have a with

Reply:
Please @AndyOGo test it it should look the same as before

Reply:
Indeed, you are right.
This additional DOM node always bothered me, that's one reason I decided not to use the `<axa-button>`.
We have to refactor buttons anyway https://github.com/axa-ch/patterns-library/issues/674

Reply:
yepp but maybe lets wait for better built in support. Anyway out of scope of pr. Please test it if u can and then we merge

Reply:
The sticky thing will be in another pr